### DISCUSSION TITLE
Multiple Databases per folder?

### DISCUSSION THREAD

**Discussion by csss1234** [RafaelGB/obsidian-db-folder#312] [Q&A] [answered]
I thought I read somewhere that only one DB per folder was allowed, but two seem to be working for me. Just checking, don't want to break anything 😅

**Comment by RafaelGB** [comment, marked as answer]
you can have all ddbb as you want its just another file of your vault. Also another ddbb are filtered from your rows so there is no problem with have more than 1

I recommend to have different sources/filters so its like to have multiple views of them

**Reply by csss1234** [reply]
Yes but if you change the header in one database, it will not automatically change in the others.

**Reply by RafaelGB** [reply]
Every ddbb as its own configuration and its totally independent of the others.

That use case requires multiple views inside the same ddbb (exist an issue on the roadmap for that)

Its a young plugin, I am sure that can arrives in a future :)
